company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise4Test.java

```diff
@@ -78,11 +80,13 @@ public void countSuppliersWithMoreThanTwoItems()
     @Test
     public void whoSuppliesSandwichToaster()
     {
+        ImmutableList<Supplier> suppliers = Lists.immutable.with(this.company.getSuppliers());
```

Comment:
Please remove this line and use `ArrayIterate` to below to iterate over supplies.